stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -719,17 +846,107 @@ public String getId() {
+    public String getAddressZipCheck() {
+        return addressZipCheck;
+    }
+
+    /**
+     * @return The ID of the customer that this card belongs to.
+     */
+    @Nullable
+    public String getCustomerId() {
+        return customerId;
+    }
+
+    /**
+     * @return If a CVC was provided, results of the check:
+     * pass, fail, unavailable, or unchecked.
+     */
+    @Nullable
+    public String getCvcCheck() {
+        return cvcCheck;
+    }
+
+    @NonNull
+    @Override
+    public JSONObject toJson() {
+        JSONObject object = new JSONObject();
+        putStringIfNotNull(object, FIELD_NAME, name);
+        putStringIfNotNull(object, FIELD_ADDRESS_CITY, addressCity);
+        putStringIfNotNull(object, FIELD_ADDRESS_COUNTRY, addressCountry);
+        putStringIfNotNull(object, FIELD_ADDRESS_LINE1, addressLine1);
+        putStringIfNotNull(object, FIELD_ADDRESS_LINE1_CHECK, addressLine1Check);
+        putStringIfNotNull(object, FIELD_ADDRESS_LINE2, addressLine2);
+        putStringIfNotNull(object, FIELD_ADDRESS_STATE, addressState);
+        putStringIfNotNull(object, FIELD_ADDRESS_ZIP, addressZip);
+        putStringIfNotNull(object, FIELD_ADDRESS_ZIP_CHECK, addressZipCheck);
+        putStringIfNotNull(object, FIELD_BRAND, brand);
+        putStringIfNotNull(object, FIELD_CURRENCY, currency);
+        putStringIfNotNull(object, FIELD_COUNTRY, country);
+        putStringIfNotNull(object, FIELD_CUSTOMER, customerId);
+        putIntegerIfNotNull(object, FIELD_EXP_MONTH, expMonth);
+        putIntegerIfNotNull(object, FIELD_EXP_YEAR, expYear);
+        putStringIfNotNull(object, FIELD_FINGERPRINT, fingerprint);
+        putStringIfNotNull(object, FIELD_FUNDING, funding);
+        putStringIfNotNull(object, FIELD_CVC_CHECK, cvcCheck);
+        putStringIfNotNull(object, FIELD_LAST4, last4);
+        putStringIfNotNull(object, FIELD_OBJECT, VALUE_CARD);
+        return object;
+    }
+
+    @NonNull
+    @Override
+    public Map<String, Object> toMap() {
+        Map<String, Object> map = new HashMap<>();
+        map.put(FIELD_NAME, name);
+        map.put(FIELD_ADDRESS_CITY, addressCity);
+        map.put(FIELD_ADDRESS_COUNTRY, addressCountry);
+        map.put(FIELD_ADDRESS_LINE1, addressLine1);
+        map.put(FIELD_ADDRESS_LINE1_CHECK, addressLine1Check);
+        map.put(FIELD_ADDRESS_LINE2, addressLine2);
+        map.put(FIELD_ADDRESS_STATE, addressState);
+        map.put(FIELD_ADDRESS_ZIP, addressZip);
+        map.put(FIELD_ADDRESS_ZIP_CHECK, addressZipCheck);
+        map.put(FIELD_BRAND, brand);
+        map.put(FIELD_CURRENCY, currency);
+        map.put(FIELD_COUNTRY, country);
+        map.put(FIELD_CUSTOMER, customerId);
+        map.put(FIELD_FINGERPRINT, fingerprint);
+        map.put(FIELD_FUNDING, funding);
+        map.put(FIELD_CVC_CHECK, cvcCheck);
+        map.put(FIELD_LAST4, last4);
+        map.put(FIELD_OBJECT, VALUE_CARD);
+        StripeNetworkUtils.removeNullAndEmptyParams(map);
```

Comment:
are we excluded those because they're integers? should we convert them to strings and include them anyway>